pages/add/form/detail-info-fields.js

```diff
@@ -1,10 +1,15 @@
 import React from 'react';
 import ReactTags from 'react-tag-autocomplete';
+import DynamicCheckboxGroup from '../../../components/form-fields/dynamic-checkbox-group.jsx';
 import validator from './validator';
 import Service from '../../../js/service';
 
 const DELIMITERS = [9,13,188]; // keycodes for tab,enter,comma
 
+let Issues = (props) => {
+  return <DynamicCheckboxGroup funcToFetchOptions={Service.issues.get} onChange={props.onChange} />;
```

Comment:
based on the comment above, why `funcToFetchOptions` instead of making Issues a stateful component?
```
class Issues extends Component {
  constructor(props) {
    super(props);
    this.state = { options: [] };
  }
  componentWillMount() {
    Service.issues.get().then(options => this.setState({ options }));
  }
  render() {
    return <DynamicCheckboxGroup options={this.state.options} onChange={this.props.onChange} />;
  }
}
```